ddpui/core/alerts/alert_service.py

```
@@ -0,0 +1,461 @@
+                query_executed=sql,
+                recipients=alert.recipients,
+                message=alert.message,
+                fired=fired,
+                rows_returned=rows_count,
+                result_preview=normalized_results[:25],
+                rendered_message=rendered_message,
+                trigger_flow_run_id=trigger_flow_run_id,
+            )
+            return fired, rows_count, rendered_message
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)